src/App.vue

```diff
@@ -33,15 +33,28 @@
   --code-line-height: 24px;
   --code-text-light: #476582;
   --code-inline-bg-color: rgba(27, 31, 35, 0.05);
+  --accent-color: #cacaca;
+  --background-color-primary:  #fff;
+  --background-color-secondary: #fafafa;
 }
+
+:root.dark-theme {
+  --background-color-primary: #1e1e1e;
+  --background-color-secondary: #2d2d30;
+  --accent-color: #3f3f3f;
+  --text-primary-color: #ddd;
```

Comment:
Where is used this color ?

Reply:
Earlier I used this accent-color for dropdown but later I used background-color-secondary. I Will remove this two in latest commit.